server/cluster/cluster.go

```diff
@@ -1590,9 +1592,12 @@ func (c *RaftCluster) checkReplicaBeforeOfflineStore(storeID uint64) error {
 	return nil
 }
 
-func (c *RaftCluster) getUpStores() []uint64 {
+func (c *RaftCluster) getUpStores(isInclude ...func(store *core.StoreInfo) bool) []uint64 {
```

Comment:
It seems that currently there’s only the usage of `IsTiKV()`. How about writing the logic directly inside the function instead of introducing the `isInclude` parameter?